puppet/zulip/files/cron.d/archive-messages

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Daily cron job for retention policy.
```

Comment:
We should probably be clear that this handles deleted messages and retention policy.

Reply:
Expanded the comment